bower.json

```diff
@@ -2,19 +2,20 @@
   "name": "styleguide",
   "version": "0.1.0",
   "dependencies": {
+    "highlightjs": "^9.2.0",
```

Comment:
`angular-highlight` no longer includes `highlightjs` as a dependency so it needs to be included explicitly.